_docs/geyser/setup.md

```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
+3. If 2. does not apply: download a jar of Geyser from the [build server](https://ci.opencollab.dev/job/Geyser/job/master/) depending on what platform your server runs on. See the [FAQ](/geyser/faq/#which-plugin-version-of-geyser-do-i-need) if you're confused about which build to download. Keep in mind: Geyser-spigot.jar is for Spigot and it's forks, such as PaperMC or Purpur. 
+4. Put the Geyser jar file in your plugins folder and start up the server.
+5. Port-forwarding: 
+ - If you are using a hosting provider, you will likely need to change your Bedrock port in `config.yml`. Information on your hosting provider might be available on the [Supported Hosting Providers](/geyser/supported-hosting-providers/) page. If there is no information on your hosting provider and you are unable to assign/open ports yourself , try enabling `clone-remote-port` in Geyser's `config.yml`file and connecting with the same IP/port as you would with Java Edition.
+ - If not using a hosting provider, you will need to use port forwarding to allow people outside your LAN to connect. You want to port forward 19132 UDP, instructions to do that for most routers can be found [here](https://www.lifewire.com/how-to-port-forward-4163829). If you are trying to join from the same LAN, the server should show up in the friends tab.
+ - To check whether it is working correctly, you can use the `geyser connectiontest [yourIP]:[yourPort]` command in your server console to check if the server is reachable from the outside.
+ - _Restart your server if you edited your config! Simply reloading will not work._
+6. Besides the `port` entry in the _Bedrock Section_ of Geyser's `config.yml`, you rarely need to change anything. A few examples:
+ - `address: 0.0.0.0` The IP address that will listen for connections. There is no reason to change this unless you want to limit what IPs can connect to your server or if your hosting provider instructs you to do so.
+  - `port: 19132` The port Bedrock players will use to connect. If the option `clone-remote-port` is set to true, this port is ignored, and you'll have to use the same port on bedrock as you would on Java to connect.
+7. The _remote_ section in the `config.yml` determines, which Java server the Bedrock players join. 
+- `address: auto` This means, that Geyser configures the server's ip, port and auth-type of the java server by itself, leave this at auto.
+- `port: 25565` This should be the port of the Java server. By default, it is set to 25565 - your hosting provider may have assigned a different port to your java server, set that here.
+8. If you wish to remove the java account requirement, set up the [Floodgate](/floodgate/setup/) plugin. This will allow Bedrock players to play on your server, without you having to use offline mode.
```

Comment:
```suggestion
- `address: auto` This means, that Geyser configures the server's IP, port and auth-type of the java server by itself, leave this at auto.
- `port: 25565` This should be the port of the Java server. By default, it is set to 25565 - your hosting provider may have assigned a different port to your Java server, set that here.
8. If you wish to remove the Java account requirement, set up the [Floodgate](/floodgate/setup/) plugin. This will allow Bedrock players to play on your server, without you having to use offline mode.
```